api/external.go

```diff
@@ -377,6 +377,8 @@ func (a *API) Provider(ctx context.Context, name string, scopes string) (provide
 		return provider.NewGoogleProvider(config.External.Google, scopes)
 	case "facebook":
 		return provider.NewFacebookProvider(config.External.Facebook, scopes)
+	case "notion":
+		return provider.NewNotionProvider(config.External.Notion)
```

Comment:
does notion's api allow one to pass scopes for oauth2?

Reply:
I've tested with `/authorize?provider=notion` endpoints and seems like it works. I saw the console logging `INSERT`. 

Notion API doesn't have scopes as of now.

Reply:
![image](https://user-images.githubusercontent.com/59365435/148051106-e8f1e512-566b-4a48-a32e-e95966cec9a7.png)